### PR TITLE
Chapter1/homework/3

### PR DESCRIPTION
**Pull request by adrianhayyj**
hw3

### CODE REVIEW

File: Src/BootCamp.Chapter/Checks.cs [hunk 1]
```diff
@@ -14,28 +15,101 @@ namespace BootCamp.Chapter
...
+            }
+
+            return userInput;
         }
 
         public static float CalculateBmi(float weight, float height)
         {
-            // To do: call your implementation. 
-            return 0;
+            if (height == 0)
```
Comment (Almantask) [inline Src/BootCamp.Chapter/Checks.cs]: Codacy Issue found: [Do not check floating point equality with exact values, use a range instead.](https://app.codacy.com/manual/Almantask/CSharp-From-Zero-To-Hero/pullRequest?prid=4868137)

**Comment by Almantask** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Issues
======
- Added 1
           

Complexity increasing per file
==============================
- Src/BootCamp.Chapter/Checks.cs  2
         
```


See the complete overview on [Codacy](https://app.codacy.com/manual/Almantask/CSharp-From-Zero-To-Hero/pullRequest?prid=4868137&bid=16142205)

**Comment by Almantask** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Complexity increasing per file
==============================
- Src/BootCamp.Chapter/Checks.cs  2
         
```


See the complete overview on [Codacy](https://app.codacy.com/manual/Almantask/CSharp-From-Zero-To-Hero/pullRequest?prid=4868137&bid=16142205)

File: Src/BootCamp.Chapter/Program.cs [hunk 2]
```diff
@@ -9,6 +9,7 @@ class Program
     {
         static void Main(string[] args)
         {
+            Checks.Demo();
```
Comment (ellmaster) [inline Src/BootCamp.Chapter/Program.cs]: This should be Lessons3.Demo();

File: Src/BootCamp.Chapter/Lesson3.cs [hunk 3]
```diff
@@ -0,0 +1,104 @@
+﻿using System;
+using System.Globalization; // NumberStyles.Float, CultureInfo.InvariantCulture
+
+class Lesson3
```
Comment (Almantask) [inline Src/BootCamp.Chapter/Lesson3.cs]: Codacy Issue found: [Move 'Lesson3' into a named namespace.](https://app.codacy.com/manual/Almantask/CSharp-From-Zero-To-Hero/pullRequest?prid=4868137)

**Comment by Almantask** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Issues
======
- Added 2
           

Complexity increasing per file
==============================
- Src/BootCamp.Chapter/Lesson3.cs  3
         
```


See the complete overview on [Codacy](https://app.codacy.com/manual/Almantask/CSharp-From-Zero-To-Hero/pullRequest?prid=4868137&bid=16142205)

File: Src/BootCamp.Chapter/Checks.cs [hunk 4]
```diff
@@ -1,6 +1,7 @@
 ﻿using System;
 using System.Collections.Generic;
 using System.Text;
+using System.Globalization; // NumberStyles.Float, CultureInfo.InvariantCulture
```
Comment (Almantask) [inline Src/BootCamp.Chapter/Checks.cs]: Codacy Issue found: [Remove this unnecessary 'using'.](https://app.codacy.com/manual/Almantask/CSharp-From-Zero-To-Hero/pullRequest?prid=4868137)

**Comment by Almantask** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Issues
======
+ Solved 6
           

Complexity increasing per file
==============================
- Src/BootCamp.Chapter/Lesson3.cs  3
         
```


See the complete overview on [Codacy](https://app.codacy.com/manual/Almantask/CSharp-From-Zero-To-Hero/pullRequest?prid=4868137&bid=16142205)

**Pull request closed without merging**